packages/docs/site/docs/main/how-to-create-a-theme-using-playground.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Build a WordPress Theme
+slug: /how-to-create-a-theme-using-playground
+---
+
+import ThisIsQueryApi from '@site/docs/\_fragments/\_this_is_query_api.md';
+
+# How to Build a WordPress Theme in WordPress Playground
+
+WordPress Playground is a powerful tool that lets you build, preview, and export WordPress themes—all in your browser or locally via `wp-now`.
+
+This guide walks you through the process step-by-step.
+
+---
+
+## 🚀 1. Creating a New Site
+
+### In the Browser
+
+1. Go to [https://playground.wordpress.net](https://playground.wordpress.net)
+2. Click **"Create New Site"**
+3. Choose WordPress and PHP versions if needed
+4. Click **"Launch"**
+
+### Locally with `wp-now`
+
+1. Install `wp-now`:
+
+    ```bash
+    npm create wp-now@latest
+    ```
+
+2. Navigate to your theme folder and run:
+
+    ```bash
+    wp-now
+    ```
+
+3. Access the local site at:
+
+    ```
+    http://localhost:8888
+    ```
+
+---
+
+## 🧰 2. Preinstall Theme-Building Tools
+
+Install the following plugins for theme development:
+
+-   [Create Block Theme](https://wordpress.org/plugins/create-block-theme/)
+-   [Gutenberg](https://wordpress.org/plugins/gutenberg/)
+-   [Theme Check](https://wordpress.org/plugins/theme-check/)
+
+Directly use playground links to load plugins: https://playground.wordpress.net/#%7B%22plugins%22:%5B%22hello-dolly%22,%22gutenberg%22%5D,%20%22login%22:%20true,%20%22landingPage%22:%20%22/wp-admin/plugins.php%22%20%7D
+
+### Install via WP Admin
+
+Navigate to:
+
+**Plugins > Add New > Search > Install > Activate**
+
+### Or install via WP-CLI
+
+```bash
+wp plugin install create-block-theme gutenberg theme-check --activate
+```
+
+## 🎨 3. Install a Starter Theme
+
+Starter themes give you a foundation to begin building your WordPress theme. There are several ways to load a starter theme in Playground:
+
+### 🧩 Option 1: Install from GitHub
+
+You can load a GitHub-hosted theme directly in WordPress Playground using a URL parameter.
+
+**Example:**
+
+https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/WordPress/blueprints/refs/heads/trunk/blueprints/install-activate-setup-theme-from-gh-repo/blueprint.json
+
+
+### 📦 Option 2: Upload from ZIP
+
+You can also upload a ZIP file of your starter theme manually:
+
+1. Go to **Appearance > Themes > Add New > Upload Theme**
+2. Click **Choose File** and select your `.zip` theme file
+3. Click **Install Now**
+4. Once installed, click **Activate**
+
```

Comment:
These built up could be solved much faster with a blueprint and that would have the advantage that it can be called repeatedly with only one click and a step.